config/clusters/2i2c/demo.values.yaml

```
@@ -20,7 +20,14 @@ jupyterhub:
           url: https://2i2c.org
   hub:
     config:
+      JupyterHub:
+        authenticator_class: cilogon
       Authenticator:
         # We do not define allowed_users here since only usernames matching this regex will be allowed to login into the hub.
         # Ref: https://jupyterhub.readthedocs.io/en/stable/api/auth.html#jupyterhub.auth.Authenticator.username_pattern
         username_pattern: '^(.+@2i2c\.org|.+@rmbl\.org|deployment-service-check)$'
+      CILogonAuthenticator:
```

Not at all a suggestive message for a typo, right?

```suggestion
      CILogonOAuthenticator:
```

---

You know, I kept thinking I verified this two or three times over and over again but I clearly did not.

---

Me too! Remember I didn't find the line adding the encrypted file to the helm chart list :| after checking a few times? 😅

---

my excuse is that we're just drifting in and out of alternate universes :D